docs/lineage/airflow.md

```diff
@@ -3,19 +3,19 @@
 There's a couple ways to get lineage information from Airflow into DataHub.
 
 
-## Using Datahub's Airflow lineage backend (recommended)
+## Using Datahub's Airflow lineage backend
 
 :::caution
 
 The Airflow lineage backend is only supported in Airflow 1.10.15+ and 2.0.2+.
```

Comment:
Does the Airflow lineage backend work for MWAA and Google Cloud Composer? Maybe we should update notes here to reflect that.